### PR TITLE
Add silver scroll

### PR DESCRIPTION
**Pull request by aleku399**

### CODE REVIEW

**Comment by aleku399** [comment]
Screen Shot 2023-09-11 at 18 46 37
Screen Shot 2023-09-11 at 18 54 43

**Comment by aleku399** [comment]
@epicallan  have rebased

File: lib/Horizon.dart [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+
+    @override
+    Widget build(BuildContext context) {
+        final DateTime currentDate = DateTime.now();
+        final TextTheme textTheme = Theme.of(context).textTheme;
+        
+
+        print("currentDate , $currentDate");
+        print("currentDate.day, $currentDate.day");
+        print("currentDate.weekday, $currentDate.weekday" );
```
Comment (epicallan) [inline lib/Horizon.dart]: Suggestion: In production code, these prints would have to be removed. I think you should remove them.

File: lib/Horizon.dart [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+            ),
+        ); 
+    }
+}
+
+const String baseAssetURL =
+    'https://dartpad-workshops-io2021.web.app/getting_started_with_slivers/';
+const String headerImage = '${baseAssetURL}assets/header.jpeg';
+
+const Map<int, DailyForecast> _kDummyData = {
```
Comment (epicallan) [inline lib/Horizon.dart]: Suggestion: This variable name sounds odd. Why not just `dummyData` or something more descriptive `weatherData` ?

Lastly prefixing a variable with an `_` makes it a private class member variable. This variable is not part of any class, so the `_` is unnecessary.

File: lib/Horizon.dart [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+  0: DailyForecast(
+    id: 0,
+    imageId: '${baseAssetURL}assets/day_0.jpeg',
+    highTemp: 73,
+    lowTemp: 52,
+    description:
+        'Partly cloudy in the morning, with sun appearing in the afternoon.',
+  ),
+  1: DailyForecast(
```
Comment (epicallan) [inline lib/Horizon.dart]: Suggestion: You have a Map with integers as keys. An array implicitly has integers as keys (indexes). Therefore you could probably simplify to an Array data structure vs. a Map.

**Pull request merged**